syspage.c

```diff
@@ -100,6 +103,8 @@ struct {
 	} maps[MAX_MAPS_NB];
 
 	mapt_entry_t entries[MAX_ENTRIES_NB];     /* General entries like syspage, kernel's elf sections and plo's elf sections */
+
+	syspage_arch_t arch;
```

Comment:
Maybe it would be better to keep simple copy of syspage

```
struct {
 syspage *syspage;
 syspage syspagebuff;
} syspage_common;
```

instead of it's fields being explicitly put?

Reply:
A `syspage_t` contains pointers to `progs` and `maps`. These parameters change dynamically while new commands are executed. So that it would be difficult to keep a `syspagebuff` in .bss section.

The maximum memory for `syspage_common.maps` and `syspage_common.progs` is allocated in .bss and only declared number of maps and progs are copied into memory points by `syspage_common.syspage` before system is launched.